### PR TITLE
SigmaDAO index

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Proposed Changes

- add fast index for DAO name queries

### CODE REVIEW

File: examples/dao/setup_sigma_dao.sql [hunk 1]
```diff
@@ -11,8 +11,12 @@ CREATE TABLE IF NOT EXISTS sigma_daos (
   asset_id BIGINT -- token id
 );
 
--- Create unique index on app_id
+-- Create indexes
 CREATE UNIQUE INDEX sigma_daos_app_id_idx ON sigma_daos (app_id);
+-- create an index to do efficient text search using `%<text>%`, it requires
+-- pg_trgm extension.
+CREATE EXTENSION pg_trgm;
+CREATE INDEX sigma_daos_dao_name_idx ON users USING gin (dao_name gin_trgm_ops);
```
Comment (ratik21) [inline examples/dao/setup_sigma_dao.sql:19]: Are there any tests/performance benchmarks for the querying we're doing using postgres?

Comment (robert-zaremba) [reply examples/dao/setup_sigma_dao.sql:19]: no, we don't have a test data. Would be good to generate some. But probably later once we are closer to production.

Comment (rmeena840) [inline examples/dao/setup_sigma_dao.sql:19]: `users`. What table is this? We don't have such a table.

**Review by rmeena840 — COMMENTED** [review]
Reviewed

**Pull request merged**